wgpu/_classes.py

```diff
@@ -1425,7 +1426,7 @@ def read_mapped(self, buffer_offset=None, size=None, *, copy=True):
         raise NotImplementedError()
 
     @apidiff.add("Replacement for get_mapped_range")
-    def write_mapped(self, data, buffer_offset=None, size=None):
+    def write_mapped(self, data, buffer_offset=None):
```

Comment:
Yeah, we can do this, because this method is not in the WebGPU spec. But we must also remove the arg from the docs below.

Reply:
Thanks for catching that. Yes, I realized that this was our function and that the size arg made no sense.